src/stores/navigated.ts

```diff
@@ -0,0 +1,31 @@
+import reactMinistore from './react-ministore'
+import scrollTopStore from './scrollTop'
+
+// Keep track of whether a programmatic scroll will occur in order to ignore events.
+let isAutoScrolling = false
+
+/**
+ * Prepares the navigatedStore to ignore scroll events that are triggered programmatically.
+ */
+export const preventScrollDetection = () => {
+  isAutoScrolling = true
+  setTimeout(() => {
+    isAutoScrolling = false
+  }, 200)
```

Comment:
Where does the 200ms come from? Is that meant to match the [duration of the browser's smooth scroll](https://stackoverflow.com/a/71417475/480608)?

Reply:
No, this was the lowest number of ms that were needed to prevent interrupting the scroll, but I believe this was a consequence from a previous attempt (based on updateThoughts, which is triggered quite often). Although giving it a single frame is enough from my testing right now, I've left a delay of 50ms in case there are still some conflicting calls.
The only way that this would affect the UX is if a user navigates to a thought and starts scrolling within 50ms, in this case, the programmatic scroll would still take precedence.